### DISCUSSION TITLE
Is C#/.NET a supported backend?

### DISCUSSION THREAD

**Discussion by chtenb** [koka-lang/koka#444] [Q&A]
I see references to it at several places, but no documentation. The `--showcs` flag doesn't seem to do anything.

**Comment by TimWhiting** [comment]
Good question. It was at one point in time, I have not tried it out though. At any rate I would probably say that the priority is basically:

- C
- WASM (comes for free because of the C backend, except a few primitives / ifdefs)
- JS
- C#

I do know that Daan has updated JS and C# external functions recently, so I would assume it works to some degree.

It is worth noting though that the test suite has only been testing compiling / runtime functionality of the C backend. (There might be e few exceptions to this rule, but I'm not aware of them other than the fact that the tests that test the type-checking and don't compile / run would be the same regardless of backend). So in other words, the C backend is the most heavily tested.
